broker/src/main/java/io/camunda/zeebe/broker/transport/commandapi/CommandApiResponseWriter.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.broker.transport.commandapi;
+
+import io.camunda.zeebe.broker.transport.ApiRequestHandler.ResponseWriter;
+import io.camunda.zeebe.transport.ServerOutput;
+import org.agrona.MutableDirectBuffer;
+
+public class CommandApiResponseWriter implements ResponseWriter {
```

Comment:
This writer doesn't write anything because the response is send asynchronously. Is that right?

It took me a while to understand this. So may be add some comments to explain.

Reply:
Yes, the response is is written later, by the engine. I agree that this is confusing and will add some comments